ruby-challenges.rb

```diff
@@ -5,32 +5,117 @@
 
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
+
+
+# Pseudo code:
+# create def method called returnMe that uses .include to return the letter found in each word of the array
+# p each letter varibles to get the expected output, p returnMe(array, let('o')ter)
+
+def returnMe (array, letter)
+  filtered_words = array.select { |word| word.include?(letter) }
+  return filtered_words
+end
 filter_letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
+p returnMe(beverages_array, filter_letter_o)
+
```

Comment:
Well done!  You have the right idea! Remember ruby has implicit return, so you can actually take out the variable  and the return variable altogether.